envcontext.js

```diff
@@ -362,6 +362,21 @@ export function isSdkmanAvailable() {
   return isSdkmanSetup;
 }
 
+/**
+ * Method to check if nvm is available.
+ */
+export function isNvmAvailable() {
+  let isNvmSetup = false;
+  const result = spawnSync(process.env.SHELL || "bash", ["-i", "-c", "nvm"], {
```

Comment:
Handle the case where nvm command is not available, but NVM_DIR is present. For example: https://github.com/AppThreat/base-images/blob/main/sle/Dockerfile.python311#L15

Reply:
In the new changes if the `nvm` command is unavailable but the `NVM_DIR` environment variable is set, the code will search within the specified NVM directory for the requested Node.js version. If a matching version is found, its path will be used. Otherwise, an error message will be displayed, prompting the user to install the desired Node.js version.